pandas/io/parquet.py

```diff
@@ -92,22 +92,18 @@ def _get_path_or_handle(
     if fs is not None:
         pa_fs = import_optional_dependency("pyarrow.fs", errors="ignore")
         fsspec = import_optional_dependency("fsspec", errors="ignore")
-        if pa_fs is None and fsspec is None:
```

Comment:
Why did this change

Reply:
The build dependency setup uncovered a bug in the recent pyarrow Filesystem implementation not raising a `ValueError` consistently if pyarrow was install but fastparquet wasn't